### DISCUSSION TITLE
[과제 추가 공지] 마지막 문제 관련 공지

### DISCUSSION THREAD

**Discussion by yoomambo** [4Moyede/HexatonClass-01#106] [00_공지사항]
![DE64C904-CB30-4C85-8C29-F344A00C2A61](https://user-images.githubusercontent.com/54018187/104477782-dd719b00-5604-11eb-8319-cf3dba5fe871.png)

@NA-DK 덕규 멘티님이 질문하셔서 답변 드립니다.

예시에 나와있는 array는 위에 filtering했던 데이터들의 모음입니다.

아마 pandas를 matplotlib로 바로 그릴 수 있는 방법이 있을거에요!

**Comment by jiwoo4424** [comment]
확인했습니다

**Reply by lsj0822** [reply]
확인했습니다
